code/modules/cooking/machinery/cooking_machines/_appliance.dm

```diff
@@ -520,12 +515,17 @@
 	new /obj/item/reagent_containers/food/snacks/badrecipe(CI.container)
 
 	// Produce nasty smoke.
-	visible_message("<span class='danger'>\The [src] vomits a gout of rancid smoke!</span>")
+	visible_message(SPAN_DANGER("[src] vomits a gout of rancid smoke!"))
 	var/datum/effect/effect/system/smoke_spread/bad/smoke = new /datum/effect/effect/system/smoke_spread/bad
 	smoke.attach(src)
 	smoke.set_up(10, 0, get_turf(src), 300)
 	smoke.start()
 
+/obj/machinery/appliance/CtrlClick(var/mob/user)
+	if(!anchored)
+		return ..()
+	attempt_toggle_power(user)
```

Comment:
CtrlClicking to toggle power isn't shared by any other object in the game, I think? Is there a more consistent method you can use here?

Reply:
The alternative is a verb, and that's clunky. The idea is that you can interact with it without ever having to open a right-click menu or window UI.

Reply:
I'd prefer it to a control that breaks conventions. You'll need to make sure it is explained to the player somewhere if we do go for this CtrlClick option. (or did I miss that somewhere already)

Reply:
It's in the changelog but I think I meant to add it to examine.

Reply:
Oh, it turns out the AI already used ctrl+click, this just adds it to non-AI mobs.